docs/src/modules/java/partials/query-syntax-reference.adoc

```diff
@@ -189,11 +186,45 @@ Results for a view query can be sorted. Use `ORDER BY` with view columns to sort
 
 If no explicit ordering is specified in a view query, results will be returned in the natural index order, which is based on the filter predicates in the query.
 
+[source,genericsql,indent=0]
 ----
-SELECT * FROM customers WHERE name = :name AND age > :min_age ORDER BY age DESC
+SELECT * FROM customers WHERE name = :name AND age > :minAge ORDER BY age DESC
 ----
 
-NOTE: Some orderings may be rejected, if the view index cannot be efficiently ordered. Generally, to order by a column it should also appear in the `WHERE` conditions.
+NOTE: Some orderings may be rejected, if the view index cannot be efficiently ordered. Generally, to order by a field it should also appear in the `WHERE` conditions.
+
+=== Aggregation
+
+==== Grouping
+
+Grouping of results based on a field is supported using `collect(*)`. Each found key leads to one returned entry, where
```

Comment:
so, there was this other PR recently that replaces the usage of the term 'support' by alternatives. 
https://github.com/akka/akka-sdk/pull/137 

```suggestion
Grouping of results based on a field is done using `collect(*)`. Each found key leads to one returned entry, where
```

But I see that we use the term quite a lot in this page. Hard to replace it in most cases, IMO.